applications/openshift/api-server/api_server_etcd_ca/rule.yml

```diff
@@ -29,13 +42,19 @@ severity: medium
 references:
     cis: 1.2.32
 
-ocil_clause: '<tt>ca</tt> is not set as appropriate for <tt>etcdClientInfo</tt>'
+ocil_clause: '{{%- if product == "ocp4" %}}<tt>ca</tt> is not set as appropriate for <tt>etcdClientInfo</tt>{{% else %}}<tt>ca</tt> is not set as appropriate for <tt>storageConfig</tt>{{%- endif %}}'
 
 ocil: |-
     Run the following command on the master node(s):
+{{%- if product == "ocp4" %}}
+    <pre>$ sudo grep ca /etc/kubernetes/static-pod-resources/configmaps/config/config.yaml</pre>
+    The output should contain something similar to:
```

Comment:
```suggestion
    The output should be similar to:
```

Reply:
I would like to keep the original language considering how it might and can change.